tests/test_runner.py

```diff
@@ -13,4 +13,4 @@ def swagger_specification():
 def test_run_parser(swagger_specification: Specification):
     actual_specification = parse('tests/data/swagger.yml')
 
-    assert swagger_specification == actual_specification
```

Comment:
just out of curiosity, why did you change the order of comparison? 🙂

Reply:
@manchenkoff pytest assert message uses left as `actual` :-)

Reply:
oh, I see, thx :D